bench/src/main/java/io/atomix/bench/AtomixBenchmark.java

```diff
@@ -62,25 +62,37 @@ public static void main(String[] args) throws Exception {
     final Namespace namespace = parseArgs(args, unknown);
     final Namespace extraArgs = parseUnknown(unknown);
     extraArgs.getAttrs().forEach((key, value) -> System.setProperty(key, value.toString()));
-    new AtomixBenchmark(namespace).start();
+
+    Atomix atomix = buildAtomix(namespace);
+    atomix.start().join();
+    LOGGER.info("Atomix listening at {}", atomix.getMembershipService().getLocalMember().address());
+
+    final ManagedRestService rest = buildRestService(atomix, namespace);
+    rest.start().join();
+    LOGGER.warn("The Atomix HTTP API is BETA and is intended for development and debugging purposes only!");
```

Comment:
why in the benchmark?

Reply:
Should probably be a warning against using benchmark nodes for anything other than benchmarking